components/omnibar/signed-in/signed-in.hbs

```diff
@@ -0,0 +1,226 @@
+                          </a>
+                        </li>
+                        <li>
+                          <a
+                            href="https://viz.stjude.cloud"
+                            class="app-title"
+                          >
+                            <img
+                              class="app-title__icon"
+                              src="{{ path '/images/icon-viz.png' }}"
+                              alt="Visualization Community Icon"
+                            />
+                            <span>
+                              Visualization Community
+                              <div class="app-subtext">Curated Visualizations</div>
+                            </span>
+                          </a>
+                        </li>
+                      </ul>
+                    </div>
+                    <div class="col-md-3 order-md-3 mega-dropdown-col">
+                      <div class="mega-menu-title">
+                        Support
+                      </div>
+                      <ul class="list-unstyled">
+                        <li>
+                          <a
+                            href="https://www.stjude.cloud/docs/"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            >Help Guides
+                            <i class="fas fa-external-link-alt"></i>
+                          </a>
+                        </li>
+                        <li>
+                          <a
+                            href="https://www.stjude.cloud/docs/faq/"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            >FAQs
+                            <i class="fas fa-external-link-alt"></i>
+                          </a>
+                        </li>
+                        <li>
+                          <a
+                            href="https://stjude.cloud/contact"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            >Contact Us</a
+                          >
+                        </li>
+                      </ul>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </li>
+
+            <li class="nav-item user-dropdown dropdown">
+              <a
+                class="nav-link dropdown-toggle profile-dropdown-toggle user-avatar"
+                href=""
+                role="button"
+                data-toggle="dropdown"
+                aria-haspopup="true"
+                aria-expanded="false"
+                title="User Info"
+              >
+                <!--
+                  <span class="user-icon"></i>
+                -->
+                <span>
+                  JM
+                  <!-- This is the user's initials, but we could add profile images later -->
```

Comment:
Are we ready to set this up to be populated via accounts rather than hardcoded (even if it is initials)?